### PR TITLE
Config to XML translation for management service configuration and cl…

### PR DESCRIPTION
**Pull request by mobasherul**
…ustered resource pool configuration

### CODE REVIEW

File: clustered/client/src/test/java/org/ehcache/clustered/client/internal/config/xml/ClusteredResourceConfigurationParserTest.java [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+  public final TestName testName = new TestName();
+
+  @Test
+  public void testTranslateClusteredResourcePoolConfiguration() {
+    ClusteredResourceConfigurationParser configTranslator = new ClusteredResourceConfigurationParser();
+    ClusteredResourcePoolImpl clusteredResourcePool = new ClusteredResourcePoolImpl();
+    Node retElement = configTranslator.unparseResourcePool(clusteredResourcePool);
+    assertThat(retElement, is(notNullValue()));
+    assertThat(retElement.getNodeName(), is("ehcache:resources"));
+    assertThat(retElement.getAttributes(), is(notNullValue()));
```
Comment (henri-tremblay) [inline clustered/client/src/test/java/org/ehcache/clustered/client/internal/config/xml/ClusteredResourceConfigurationParserTest.java]: Please extract the result of getAttributes to a variable

File: management/src/test/java/org/ehcache/management/registry/ManagementRegistryServiceConfigurationParserTest.java [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    DefaultManagementRegistryConfiguration defaultManagementRegistryConfiguration =
+      new DefaultManagementRegistryConfiguration().setCacheManagerAlias("my-cache-alias").
+        setCollectorExecutorAlias("my-executor");
+
+    Node retElement = configTranslator.unparseServiceCreationConfiguration(defaultManagementRegistryConfiguration);
+    assertThat(retElement, is(notNullValue()));
+    assertThat(retElement.getNodeName(), is("mgm:management"));
+    assertThat(retElement.getAttributes(), is(notNullValue()));
+    assertThat(retElement.getAttributes().getLength(), is(3));
+    assertThat(retElement.getAttributes().item(0).getNodeName(), is("cache-manager-alias"));
```
Comment (henri-tremblay) [inline management/src/test/java/org/ehcache/management/registry/ManagementRegistryServiceConfigurationParserTest.java]: You could extract a method that asserts on items

File: management/src/test/java/org/ehcache/management/registry/ManagementRegistryServiceConfigurationParserTest.java [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    assertThat(retElement.getAttributes().item(0).getNodeValue(), is("my-cache-alias"));
+    assertThat(retElement.getAttributes().item(1).getNodeName(), is("collector-executor-alias"));
+    assertThat(retElement.getAttributes().item(1).getNodeValue(), is("my-executor"));
+    assertThat(retElement.getAttributes().item(2).getNodeName(), is("xmlns:mgm"));
+    assertThat(retElement.getAttributes().item(2).getNodeValue(), is("http://www.ehcache.org/v3/management"));
+    Node tagsElement = retElement.getFirstChild();
+    assertThat(tagsElement, is(notNullValue()));
+    assertThat(tagsElement.getNodeName(), is("mgm:tags"));
+    assertThat(tagsElement.getAttributes().getLength(), is(0));
+    Node tag1 = tagsElement.getFirstChild();
```
Comment (henri-tremblay) [inline management/src/test/java/org/ehcache/management/registry/ManagementRegistryServiceConfigurationParserTest.java]: You can extract a method to checking tags

File: clustered/client/src/test/java/org/ehcache/clustered/client/internal/config/xml/ClusteredResourceConfigurationParserTest.java [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+    assertThat(retElement.getAttributes().getLength(), is(1));
+    assertThat(retElement.getAttributes().item(0).getNodeName(), is("xmlns:ehcache"));
+    assertThat(retElement.getAttributes().item(0).getNodeValue(), is("http://www.ehcache.org/v3"));
+    Node clusterElement = retElement.getFirstChild();
+    assertThat(clusterElement, is(notNullValue()));
+    assertThat(clusterElement.getAttributes(), is(notNullValue()));
+    assertThat(clusterElement.getAttributes().getLength(), is(3));
```
Comment (henri-tremblay) [inline clustered/client/src/test/java/org/ehcache/clustered/client/internal/config/xml/ClusteredResourceConfigurationParserTest.java]: A bunch of method extractions will be possible here as well.

**Comment by albinsuresh** [comment]
test this please

**Comment by mobasherul** [comment]
test this please

**Pull request closed without merging**